src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -1249,14 +1251,17 @@ public CompactionInfo getCompactionInfo()
         Int2ObjectHashMap<RedundantBefore> redundantBefores = new Int2ObjectHashMap<>();
         Int2ObjectHashMap<DurableBefore> durableBefores = new Int2ObjectHashMap<>();
         Int2ObjectHashMap<RangesForEpoch> ranges = new Int2ObjectHashMap<>();
-        AsyncChains.getBlockingAndRethrow(node.commandStores().forEach(safeStore -> {
-            synchronized (redundantBefores)
-            {
-                redundantBefores.put(safeStore.commandStore().id(), safeStore.redundantBefore());
-                ranges.put(safeStore.commandStore().id(), safeStore.ranges());
-                durableBefores.put(safeStore.commandStore().id(), safeStore.durableBefore());
-            }
-        }));
+        if (node.commandStores().all().length > 0)
```

Comment:
in my tests we are testing journal but we don't have any tables marked for accord, so we had 0 stores... compaction tests were also impacted as we could have data but due to us dropping accord we no longer have stores